README.md

```diff
@@ -347,6 +371,12 @@ Wassim Chegham
 Topics: Angular, JavaScript, Progressive Web Apps, Chatbots, Firebase, Cloud, Internet of Things  
 https://twitter.com/manekinekko
 
+<img src="https://pbs.twimg.com/profile_images/760590433508585472/4062rqec_bigger.jpg" height="70px" width="auto" align="left" alt="" />
+
+Curtis "Ovid" Poe  
+Topics: Perl, Perl6, Relational Model (SQL, ORMs), Good Development Practices
+https://twitter.com/OvidPerl
```

Comment:
I now live in Cannes, France. I'd also add "Agile Training" and "Enterprise Testing" to that list.